_connect-files/api/objects/form-properties/sources/saas/klaviyo-object.md

```diff
@@ -0,0 +1,51 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+## PLEASE REMOVE COMMENTS WHEN FINISHED
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-klaviyo-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "Klaviyo Source Form Property"
+api-type: "platform.klaviyo"
+display-name: "Klaviyo"
+
+source-type: "saas"
+docs-name: "klaviyo"
+
+property-description: ""
+## Used to create a description for the object that doesn't adhere to the standard in _developers/connect/api/documentation/api-form-properties.html
+## See the Heap object for an example
+
+description: ""
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+uses-start-date: true
+
+# object-attributes:
```

Comment:
There's actually a property users need to provide to create this source using the API.